src/Microsoft.DotNet.Build.Tasks/Targets/test-runtime/project.json

```diff
@@ -8,12 +8,12 @@
       "OpenCover": "4.6.166",
       "ReportGenerator": "2.3.1",
 
-      "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0014",
-      "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0014",
+      "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0022",
+      "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0022",
 
-      "xunit": "2.1.0-rc1-build3168",
-      "xunit.console.netcore": "1.0.2-prerelease-00088",
-      "xunit.runner.utility": "2.1.0-rc1-build3168"
+      "xunit": "2.1.0",
+      "xunit.console.netcore": "1.0.2-prerelease-00100",
```

Comment:
Might need to modify this. I want the project.json to reference the xunit.console.netcore package that will be built when this PR merges.